### PR TITLE
Add dependencies

### PR DESCRIPTION
**Pull request by daoneandonly**
Hey team!

I've added the basics of a server. Installing `express` and `nodemon`, both of which you should be familiar with.
Now before adding this to the `develop` branch, I want to ask what you want to use as the **templating engine**.

Any preferences for EJS, Handlebars or Pug?

### CODE REVIEW

**Comment by pepijnkok** [comment]
Hey,

I prefer using EJS because I got no experience with the other 2.

File: package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "description": "A2 assignment for Backend at CMDA",
   "main": "app.js",
   "scripts": {
+    "start": "node app.js",
+    "dev": "nodemon ./server.js localhost 3000",
```
Comment (pepijnkok) [inline package.json:8]: Looks good! I would suggest to put this information in the readme aswell.

Comment (daoneandonly) [reply package.json:8]: Thanks. We will add readme changes later 👍

File: views/pages/index.ejs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<%- include('../partials/head'); %>
+<%- include('../partials/header'); %>
```
Comment (pepijnkok) [inline views/pages/index.ejs:2]: Lets be consistent with the usage of ' ; ' as a team

**Pull request merged**